app/forms/ReceiveGoodsNIPForm.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.FormValidation._
+import models.ConditionalValue
+import play.api.data.Form
+import play.api.data.Forms._
+import uk.gov.voa.play.form.ConditionalMappings._
+
+object ReceiveGoodsNIPForm extends RequiredBooleanForm {
+  override val errorMsg = "validation.northernIrelandReceiveGoods.missing"
+  val INPUT_NORTHERN_IRELAND_AMOUNT: String = "northernIrelandReceiveGoods"
```

Comment:
I wouldn't use all caps for val names, that's not our pattern but an old vrs one